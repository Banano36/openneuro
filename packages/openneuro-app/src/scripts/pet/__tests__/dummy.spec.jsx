import React from 'react'
import { shallow } from 'enzyme'
import PETDummy from '../dummy.jsx'

jest.mock('openneuro-content')

describe('pet/dummy.jsx', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<PETDummy />)
    expect(wrapper).toMatchSnapshot()
  })
})
