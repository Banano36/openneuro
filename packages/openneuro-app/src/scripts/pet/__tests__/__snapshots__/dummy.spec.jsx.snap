// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pet/dummy.jsx renders successfully 1`] = `
<span
  className="front-page is-front"
>
  <div
    className="intro pet-block"
  >
    <div
      className="container"
    >
      <div
        className="intro-inner fade-in clearfix"
      >
        <div
          className="clearfix welcome-block"
        >
          <h1
            className="pet-header"
          >
            Coming soon!
          </h1>
          <h2>
            A section of OpenNeuro dedicated entirely to PET scans is currently under development.
             
          </h2>
          <h3>
            In the meantime, feel free to explore
             
            <Link
              to="/"
            >
              OpenNeuro
            </Link>
             or visit existing hosted PET datasets
             
            <Link
              to="/datasets/ds001421"
            >
              [11C]SB207145 PET Cimbi database example
            </Link>
            ,
             
            <Link
              to="/datasets/ds001420"
            >
              [11C]DASB PET Cimbi database example
            </Link>
            , or
             
            <Link
              to="/datasets/ds000204"
            >
              Imaging [18F]AV-1451 and [18F]AV-45 in acute and chronic traumatic brain injury
            </Link>
            .
          </h3>
          <div
            className="privacy-detail"
          >
            <span />
          </div>
        </div>
      </div>
    </div>
  </div>
  <Footer />
</span>
`;
